### DISCUSSION TITLE
[Feature] edit, crop, draw, add text

### DISCUSSION THREAD

**Discussion by dkerlee** [immich-app/immich#9373] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I need the ability to crop, rotate, draw on, and add text right on pictures.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of https://github.com/immich-app/immich/issues/768

**Reply by dkerlee** [reply]
Thank you. That was why I didn’t find it, I was searching Active feature requests.

So: if Alex has marked this feature request as something not within the scope of the project, are you no longer accepting votes for this feature request?

That feels like a not-listening-to-community-feedback. Even though Alex said they’re not doing it, I still want to be able to vote and ask for the feature.

**Reply by schuhbacca** [reply]
Did you read the entire thread?
